DEPS

```diff
@@ -368,6 +374,9 @@ deps = {
   'src/third_party/colorama/src':
    Var('chromium_git') + '/external/colorama.git' + '@' + '799604a1041e9b3bc5d2789ecbd7e8db2e18e6b8',
 
+   'src/third_party/emsdk':
+   Var('skia_git') + '/external/github.com/emscripten-core/emsdk.git' + '@' + 'fc645b7626ebf86530dbd82fbece74d457e7ae07',
```

Comment:
It may be easier to put this under something like `src/buildtools` or just `src/emsdk`. We'll also want to exclude wherever this ends up from the license bot. Generally though, third_party is source code rather than build tools.

Reply:
Done